### ISSUE TITLE
Stub method with a docstring isn't a stub anymore?

### ISSUE THREAD

**Issue by MatejNikl** [nhoad/flake8-unused-arguments#3]
Hey, thanks for this nice plugin! I have the following suggestion for improvement:

Consider this code snippet:
```python
class Base:
    def foo(self, value):
        """To be (optionally) reimplemented by childen."""
        pass
```
Even though I would consider the `foo` method to be a stub and have the `unused-arguments-ignore-stub-functions` option enabled, the plugin still reports the `value` variable as unused.

Removing the comment helps, i.e. the plugin correctly doesn't produce the warning for the following snippet:
```python
class Base:
    def fn(self, an_arg: int):
        pass
```

Would it be possible to consider methods with a docstring (and having otherwise empty body) as stubs as well?

**Comment by nhoad** [comment]
Hi! Thank you, I'm glad you like it :) that's a good idea - I've just released 0.0.6 that does that. It will also consider functions that _only_ contain a docstring to be stubs as well.

Thank you for the suggestion!
